modules/admin-ui/src/main/resources/web/admin/apps/content-manager/js/app/create/NewContentDialog.ts

```diff
@@ -60,8 +65,17 @@ module app.create {
             });
 
             this.contentList = new app.create.NewContentDialogList();
+            this.contentList.addClass("new-content-types-list");
```

Comment:
'content-types-list' is also used in 'Recently Used' block, I added new class 'new-content-types-list' to distinguish between main area of content types and recently used block